style/style.css

```diff
@@ -0,0 +1,55 @@
+/*
+    ==============================================
+                ****** RESET ******
+    ==============================================
+*/
+body {
+  margin: 0;
+}
+
+:root {
+  --color-red: #f42;
+  --bg-color-red: #f42;
+}
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+a:hover {
+  color: #333;
+}
+
+/*
+    ==============================================
+                ****** GLOBAL ******
+    ==============================================
+*/
+
+#nav-bar {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+}
+
+@media screen and(max-width: 768px) {
+  #nav-bar {
+    display: flex;
+  }
+}
```

Comment:
Great used media query ! 😄

Reply:
There was a great use of the **`requirement checklist`**. I think it would have been great if you highlighted in the inline comments the changes the `README `required as well. That would have been great as it would have made the review complete. There were some areas that needed highlighting in the `README`, like filling out the placeholders with actual content describing the project. Good job in general. Everything else looked professional.,